articles/hooks/extensibility-points/send-phone-message.md

```diff
@@ -0,0 +1,108 @@
+---
+description: Learn how hooks can be used with the Send Phone Message extensibility point, which is available for MFA.
+beta: true
+toc: true
+topics:
+    - hooks
+    - extensibility-points
+    - send-phone-message
+    - custom-sms-gateway
+    - sms 
+contentType:
+  - how-to
+useCase: extensibility-hooks
+v2: true
+---
+# Send Phone Message
+
+At the Send Phone Message extensibility point, Hooks allow custom actions to be executed when a user is prompted for MFA. For example, you may use your preferred SMS gateway provider to get better pricing.
+
+The Send Phone Message extensibility point is available for [MFA](/multifactor-authentication) requests.
+
+::: note
+The `triggerId` for the Send Phone Message extensibility point is `send-phone-message`. To learn how to create Hooks for this extensibility point, see [Create New Hooks](/hooks/create).
+:::
```

Comment:
This should be mentioned at the end of the document. How is the 'send-phone-message' triggerID relevant for customers ? If they want to create the hook using the Management API? If so, is there a document where we explain how to do that?

Reply:
It is explained on the linked page: https://auth0.com/docs/hooks/create . Or have I misunderstood?

Reply:
Right, I missed the 'Management API' tab in the document. Let's move this to the bottom in a section with the title "Configuring the hook using the Management API"

Reply:
Got it, pushed this change.